projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-row.component.ts

```diff
@@ -93,6 +94,20 @@ export class IgxHierarchicalRowComponent extends IgxRowComponent<IgxHierarchical
         });
     }
 
+    private endEdit(grid: IgxHierarchicalGridComponent) {
+        while (grid.parent) {
```

Comment:
There is already a getter for the rootGrid that you can use --> grid.rootGrid

Reply:
thank you
changed :)